docs/other/troubleshooting-gratia.md

```diff
@@ -0,0 +1,375 @@
+
+[Site Information]
+; The group option indicates the group that the OSG site should be listed in,
+; for production sites this should be OSG, for vtb or itb testing it should be
+; OSG-ITB
+; 
+; YOU WILL NEED TO CHANGE THIS
+group = OSG
+
+; The host_name setting should give the host name of the CE  that is being 
+; configured, this setting must be a valid dns name that resolves
+; 
+; YOU WILL NEED TO CHANGE THIS
+host_name = tusker-gw1.unl.edu
+
+; The resource setting should be set to the same value as used in the OIM 
+; registration at the goc 
+; 
+; YOU WILL NEED TO CHANGE THIS
+resource = Tusker-CE1
+
+; The resource_group setting should be set to the same value as used in the OIM 
+; registration at the goc 
+; 
+; YOU WILL NEED TO CHANGE THIS
+resource_group = Tusker
+
+```
+
+Do those names match the names that you registered with OIM? If not, edit the names, and rerun "osg-configure -c".
+
+Did the site name change?
+-------------------------
+
+Was the site previously reporting data, but the site name (not host name, but site name) changed? When the site name changes, you need to ask the Gratia operations team to update the name of your site at the GRACC collector. To do this:
+
+1.  Open a ticket at [the GOC ticket web page](https://ticket.grid.iu.edu/goc/submit)
+2.  Select "Software or Service"
+3.  Select "GRACC Operations"
+4.  Type a friendly email that asks the GRACC team to change your site name at the collector. Make sure to tell them the old name and the new name.
+
+ Is a site reporting data?
+--------------------------
+
+You can see if the OSG GRACC Server is getting data from a site by going to [GRACC](https://gracc.opensciencegrid.org/dashboard/db/pilot-jobs-summary?orgId=1):
+
+1.  Specify the site name in Facility
+
+Condor must be configured to put information about each job into a special directory.
+-------------------------------------------------------------------------------------
+
+Gratia will read and remove the files in order to collect the accounting information.
+
+The configuration variable is called `PER_JOB_HISTORY_DIR`. If you install the OSG RPM for Condor, the Gratia probe will extend its configuration by adding a file to `/etc/condor/config.d`, and will set this variable to `/var/lib/gratia/data`. If you are using a different installation method, you probably need to set the variable yourself. You can check if it's set by using `condor_config_val`, like this:
+
+    :::console
+    user@host $ condor_config_val -v PER_JOB_HISTORY_DIR
+    PER_JOB_HISTORY_DIR: /var/lib/gratia/data
+        Defined in '/etc/condor/config.d/99_gratia.conf', line 5.
+
+If you set this value, you need to restart condor:
+
+    :::console
+    root@host # condor_restart
+    Sent "Restart" command to local master
+
+Unlike many Condor settings, a **condor\_reconfig** is not sufficient - you must restart!
+
+If you accidentally did not set `PER_JOB_HISTORY_DIR` (see above)
```

Comment:
Make this a level three heading and call it `Condor history recovery` or something